internal/generator/suite.go

```diff
@@ -191,3 +193,68 @@ func (s *Suite) String() string {
 
 	return spaceRegex.ReplaceAllString(strings.TrimSpace(result.String()), "\n")
 }
+
+const bashSuiteTemplate = `
+function setup() {
+	{{ .Setup }}
+}
+
+function cleanup() {
+	{{ .Cleanup }}
+}
+`
+
+// BashString generates bash script for the suite
+func (s *Suite) BashString() string {
+	setup := strings.Join(s.getCompleteSetup(), "\n")
+	cleanup := strings.Join(s.getCompleteCleanup(), "\n")
```

Comment:
If setup or cleanup is empty, generated bash file will be invalid, because bash functions should always have a body.
We can insert a dummy body to avoid error when running generated scripts.
```suggestion
	setup := strings.Join(s.getCompleteSetup(), "\n")
	cleanup := strings.Join(s.getCompleteCleanup(), "\n")

	if setup == "" {
		setup = ":"
	}
	if cleanup == "" {
		cleanup = ":"
	}
```